src/extension_commands.ml

```diff
@@ -131,6 +133,131 @@ let _open_current_dune_file =
   in
   command Extension_consts.Commands.open_current_dune_file handler
 
+let _jump_to_prev_hole, _jump_to_next_hole =
+  let pick_next_hole current_pos (`Sorted_non_empty (holes : Range.t list)) :
```

Comment:
I'm not sure how useful a `` `Sorted_non_empty`` tag is.
It's still possible to call it with an empty list: `` `Sorted_non_empty []``

A `non_empty_list` type would be typesafe, but might be too much for a few functions.

Maybe it would be better to have it as a label name? `~sorted_non_empty_holes`
It's not any more typesafe than the polymorphic variant, but it would have to be present at the calling site.